File: scraper/src/entities/bank_rate.py
class BankRate:
    def __init__(self, bank_id: str, buy: float, sell: float) -> None:
        self.bank_id = bank_id
        self.buy = buy
        self.sell = sell


class DollarRate(BankRate):
    pass


class EuroRate(BankRate):
    pass

File: scraper/src/entities/bankRate.ts
import { BANK } from "src/enums";

export class BankRate {
  bank: BANK;
  buy: number;
  sell: number;

  constructor(bank: BANK, buy: number, sell: number) {
    this.bank = bank;
    this.buy = buy;
    this.sell = sell;
  }
}

export class DollarRate extends BankRate {}

export class EuroRate extends BankRate {}
